File: URI Online Judge - Beecrowd/Beginner/1036.c
#include <stdio.h>
 
int main() {
 double bhask1,baixo, bhask2,a ,b ,c, delta;
	scanf("%lf", &a);
	scanf("%lf", &b);
	scanf("%lf", &c);
	
	baixo = 2*a;
	delta = b*b-4*a*c;
	
	bhask1 = (-b+sqrt(delta))/(baixo);
	
	bhask2 = (-b-sqrt(delta))/(baixo);
	
	if((delta < 0) || (baixo <= 0)){
		printf("Impossivel calcular\n");
	}else {
		printf("R1 = %.5lf\n",bhask1);
		printf("R2 = %.5lf\n",bhask2);
	}
	return 0;
}

File: URI Online Judge - Beecrowd/Beginner/1036.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var a, b, c float64
	var total, r1, r2 float64
	fmt.Scanf("%f %f %f", &a, &b, &c)

	total = math.Pow(b, 2) - 4*a*c

	r1 = (-b + math.Sqrt(total)) / (2 * a)
	r2 = (-b - math.Sqrt(total)) / (2 * a)

	if a == 0 || total <= 0 {
		fmt.Printf("Impossivel calcular\n")
	} else {
		fmt.Printf("R1 = %.5f\n", r1)
		fmt.Printf("R2 = %.5f\n", r2)
	}

}
